apps/keyboard/js/imes/latin/latin.js

```diff
@@ -192,7 +189,7 @@
           default:
             return (type === 'textarea') ? 'latin-prose' : 'latin';
         }
-
+      /* falls through */
```

Comment:
thought it's strange, it does applied here. add `break` will cause `return before executing break statement` error :/